<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainexewindow</class>
 <widget class="QDialog" name="mainexewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(51, 51, 51, 255), stop:				0.525822 rgba(95, 95, 95, 255), stop:1 rgba(58, 58, 58, 255));
}

QFrame {
	background-color: rgb(75, 75, 75);
	border: 6px groove #4c4c4c;
	border-radius: 20px;
}

QLabel {
	background: transparent;
	border: none;
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;
	color: #dadada;
}

QPushButton {
	height: 70px;
	width: 265px;
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;	
	color: #dadada;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.431818, x2:1, y2:0.409364, stop:0.217877 			rgba(43, 	43, 43, 255), stop:0.50838 rgba(64, 64, 64, 255), stop:0.938547 rgba(37, 37, 37, 255));
	border: 2px groove #4a8390;
	border-radius: 10px;
	padding: 6px;
	text-align: justify;
}

QPushButton:hover {
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.488, y2:0, stop:0 rgba(37, 37, 37, 255), 	stop:1 rgba(104, 104, 104, 255));
}

QPushButton:pressed {
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.488, y2:0, stop:0 rgba(22, 22, 22, 255), stop:1 rgba(64, 64, 64, 255));
}

QLabel#pirkkal {	
	color: #dadada;
	font: 350 10pt &quot;Leelawadee UI Semilight&quot;;
	qproperty-geometry:rect(610 550 101 35);
}

QLabel#btnsulje:hover {
	color: rgb(247, 95, 97);
}

</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>149</y>
     <width>571</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnSaldo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>                 Saldo</string>
      </property>
      <property name="icon">
       <iconset resource="gifsANDicons.qrc">
        <normaloff>:/icons/icons/piggy-bank.png</normaloff>:/icons/icons/piggy-bank.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnTilisiirto">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>             Tilisiirto</string>
      </property>
      <property name="icon">
       <iconset resource="gifsANDicons.qrc">
        <normaloff>:/icons/icons/transfer-money.png</normaloff>:/icons/icons/transfer-money.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnOtto">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>                  Otto</string>
      </property>
      <property name="icon">
       <iconset resource="gifsANDicons.qrc">
        <normaloff>:/icons/icons/withdraw.png</normaloff>:/icons/icons/withdraw.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnKirjauduulos">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>       Kirjaudu ulos</string>
      </property>
      <property name="icon">
       <iconset resource="gifsANDicons.qrc">
        <normaloff>:/icons/icons/log-out.png</normaloff>:/icons/icons/log-out.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnTapahtumat">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>       Tapahtumat</string>
      </property>
      <property name="icon">
       <iconset resource="gifsANDicons.qrc">
        <normaloff>:/icons/icons/transaction.png</normaloff>:/icons/icons/transaction.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnsulje">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>        Sulje ikkuna</string>
      </property>
      <property name="icon">
       <iconset resource="gifsANDicons.qrc">
        <normaloff>:/icons/icons/close-button.png</normaloff>:/icons/icons/close-button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>631</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pirkkal">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>101</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Pirkka Pankki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>551</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Valitse haluamasi toiminto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pirkkal</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="gifsANDicons.qrc"/>
 </resources>
 <connections/>
</ui>
